libminifi/src/io/StreamFactory.cpp

```diff
@@ -16,12 +16,13 @@
  * limitations under the License.
  */
 #include "io/StreamFactory.h"
+
 #include <atomic>
 #include <memory>
 #include <mutex>
 #include <string>
-#include <utils/GeneralUtils.h>
 
+#include <utils/GeneralUtils.h> // NOLINT
```

Comment:
why do we need NOLINT here?
Also, end of line comments should have 2 preceding spaces.

Reply:
This is incorrectly recognized as a C header. Also `NOLINT` comments do not need to be preceded by 2 spaces.

Reply:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

I think if we replaced the angle brackets with quotation marks, the warning would go away. Could you try this? If it doesn't work, them I'm fine with leaving the NOLINT comments of such cases in place.